docs/getting_started/tutorial_quickstart.md

```diff
@@ -1,3 +1,183 @@
+
+```{note}
+TODO: link to a list of included kernels
+```
+
+```{code-cell}
+kernels = [parcels.kernels.AdvectionEE]
+```
+
+## Prepare output: `ParticleFile`
+
+Before starting the simulation, we must define where and how frequent we want to write the output of our simulation. We can define this in a `ParticleFile` object:
+
+```{code-cell}
+output_file = parcels.ParticleFile("output-quickstart.zarr", outputdt=np.timedelta64(1, "h"))
+```
+
+The output files are in `.zarr` [format](https://zarr.readthedocs.io/en/stable/), which can be read by `xarray`. See the [Parcels output tutorial](./tutorial_output.ipynb) for more information on the zarr format. We want to choose the `outputdt` argument so that it captures the smallest timescales of our interest.
+
+## Run Simulation: `ParticleSet.execute()`
+
+Finally, we can run the simulation by _executing_ the `ParticleSet` using the specified list of `kernels`. Additionally, we need to specify:
+
+- the `runtime`: for how long we want to simulate particles.
+- the `dt`: the timestep with which to perform the numerical integration in the `kernels`. Depending on the numerical integration scheme, the accuracy of our simulation will depend on `dt`. Read [this notebook](https://github.com/Parcels-code/10year-anniversary-session2/blob/8931ef69577dbf00273a5ab4b7cf522667e146c5/advection_and_windage.ipynb) to learn more about numerical accuracy.
+
+```{note}
+TODO: add Michaels 10-years Parcels notebook to the user guide
+```
+
+```{code-cell}
+:tags: [hide-output]
+pset.execute(
+    kernels,
+    runtime=np.timedelta64(1, "D"),
+    dt=np.timedelta64(5, "m"),
+    output_file=output_file,
+)
+```
+
+## Read output
+
+To start analyzing the trajectories computed by **Parcels**, we can open the `ParticleFile` using `xarray`:
+
+```{code-cell}
+ds_particles = xr.open_zarr("output-quickstart.zarr")
+ds_particles
+```
+
+The 10 particle trajectories are stored along the `trajectory` dimension, and each trajectory contains 25 observations (initial values + 24 hourly timesteps) along the `obs` dimension. The [working with Parcels output tutorial](./tutorial_output.ipynb) provides more detail about the dataset and how to analyse it.
+
+Let's verify that Parcels has computed the advection of the virtual particles!
+
+```{code-cell}
+import matplotlib.pyplot as plt
+
+# plot positions and color particles by number of observation
+scatter = plt.scatter(ds_particles.lon.T, ds_particles.lat.T, c=np.repeat(ds_particles.obs.values,npart), cmap="Greys", edgecolors='r')
```

Comment:
I meant only the starting positions with a red circle. I liked the previous colormap for all the other positions. So that it's clearer where the start positions are